website/docs/guides/1.1.0-upgrade-guide.html.markdown

```diff
@@ -8,73 +8,87 @@ MongoDB Atlas Provider 1.1.0: Upgrade and Information Guide
 
-### Option 1:  Remove the existing and import the new resource
+1.1.0 also includes general improvements and bug fixes. See the [CHANGELOG](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md) for more specific information.
 
-1. Remove your existing `mongodbatlas_cloud_provider_snapshot` resources from terraform state
-   **NOTE** before doing any terraform state change please do a backup
+## Migration from Cloud Provider Snapshot to Cloud Backup Snapshot
 
-```bash
-$ terraform state rm mongodbatlas_cloud_provider_snapshot.test
-```
+The term Cloud Provider has been deprecated in favor of the term Cloud Backup.  In order to support this change within the Terraform provider, related resources and data sources with the older terminology have been deprecated and new resources and data sources added to support the new terminology.  In order to upgrade from the previous resources to the new you'll need to select from one of the following two options:
 
-2. Change the version of provider to the v1.1.0
+### Option 1: Replace the name of resource in terraform state
 
-3. In order to import the new resource, you must first change your Terraform configuration file, replacing `mongodbatlas_cloud_provider_snapshot_backup_policy` with `mongodbatlas_cloud_backup_schedule`. See the example shared above or find more information in the [mongodbatlas_cloud_backup_schedule](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cloud_backup_schedule)
+1. Replace the name of the deprecated provider `mongodbatlas_cloud_provider_snapshot` with the new name `mongodbatlas_cloud_backup_snapshot` in the Terraform state file (`terraform.tfstate`).  Do the same for `mongodbatlas_cloud_provider_snapshot_restore_job` and `mongodbatlas_cloud_backup_snapshot_restore_job`
+   **NOTE** Before performing any Terraform state change please backup your state.
 
-4. Perform an import of your existing `mongodbatlas_cloud_provider_snapshot` to `mongodbatlas_cloud_backup_snapshot`
-```bash
-    terraform import mongodbatlas_cloud_backup_snapshot.test <project_id>-<cluster_name>-<snapshot_id>
-```
+2. Upgrade the provider to v1.1.0
 
-5. Do a terraform plan (no changes detected)
+3. Run a `terraform plan` to ensure no changes are detected
 
 ```bash
 terraform plan
 
 (...)
 
-
 No changes. Your infrastructure matches the configuration.
 
 Terraform has compared your real infrastructure against your configuration and found no differences,
 so no changes are needed.
 ```
 
-### Option 2: Replace the name of resource in terraform state
+### Option 1:  Remove the existing and import the new resource
```

Comment:
choose deprecated or removed based on if the old resources are removed
```suggestion
### Option 2:  Remove the deprecated|removed and import the new resources
```

Reply:
Going with deprecated.